src/components/donation/Steps/Splits/Splits.tsx

```diff
@@ -1,37 +1,25 @@
 import { humanize } from "helpers";
-import { useEffect, useState } from "react";
-import { type SplitsStep, setSplit, setStep } from "slices/donation";
-import { useSetter } from "store/accessors";
+import { useState } from "react";
+import { useDonationState } from "../Context";
 import BackBtn from "../common/BackBtn";
 import ContinueBtn from "../common/ContinueBtn";
-import type { Config } from "../types";
+import type { SplitsStep } from "../types";
 import { LockedSplitSlider } from "./LockedSplitSlider";
 
-type Props = SplitsStep & {
-  widgetConfig: Config | null;
-};
+type Props = SplitsStep;
+export default function Split(props: Props) {
+  const { details, init, liquidSplitPct: liquidSplitFromSplitStepPct } = props;
 
-export default function Split({
-  details,
-  liquidSplitPct: donationLiquidSplit = 50,
-  widgetConfig,
-}: Props) {
-  const dispatch = useSetter();
+  const persistedLiqSplitPct =
+    init.mode === "live"
```

Comment:
if mode is `live` whatever user sets (can override initial config) on  the donate form is used, otherwise (in preview) config is persisted